.github/workflows/ci.yaml

```diff
@@ -16,6 +16,13 @@ on:
     branches: [ main ]
   workflow_call: # allow this workflow to be called by other workflows
 
+concurrency:
+  # On master/release, we don't want any jobs cancelled so the sha is used to name the group
+  # On PR branches, we cancel the job if new commits are pushed
+  # More info: https://stackoverflow.com/a/68422069/253468
```

Comment:
looks like there a newer, easier to understand answer to this now: https://stackoverflow.com/a/70972844

Reply:
Just bumping this, so we can merge?